sql/query/expr/like_test.go

```diff
@@ -0,0 +1,51 @@
+package expr
+
+import "testing"
+
+func Test_like(t *testing.T) {
+	type args struct {
+		text    string
+		pattern string
+	}
+	tests := []struct {
+		name    string
+		args    args
+		want    bool
+		wantErr bool
+	}{
+		{"Empty", args{"", ""}, true, false},
+		{"Empty false", args{"abc", ""}, false, false},
+
+		{"Any", args{"abc", "%"}, true, false},
+		{"Any false", args{"", "%"}, true, false},
+
+		{"Any one and more", args{"abc", "_"}, true, false},
```

Comment:
While porting `icuLikeCompare` I also found that this should be `false` (at least in SQLite, not sure about CockroachDB)
See also https://github.com/genjidb/genji/pull/241#discussion_r502796458